client/blocks/comment-detail/index.jsx

```diff
@@ -109,6 +109,16 @@ export class CommentDetail extends Component {
 		setCommentStatus( commentId, postId, 'trash' === commentStatus ? 'approved' : 'trash' );
 	}
 
+	keyHandler = event => {
```

Comment:
@kwight I can confirm that, as suggested by @gwwar, by simply adding this early return here is enough to fix the comment actions propagation.
```es6
if ( this.state.isExpanded ) {
  return;
}
```